source/ID/concordium-did.rst

```diff
@@ -0,0 +1,661 @@
+          },
+          "...",
+          {
+            "id": "did:ccd:NET:cred:CRED#key-N",
+            "type": "Ed25519VerificationKey2020",
+            "controller": "did:ccd:NET:cred:CRED",
+            "publicKeyMultibase": "fYY"
+          }
+        ]
+      }
+    ],
+    "authentication": [
+      "#credential-1"
+    ]
+  }
+
+
+Smart Contract Instance DID
+---------------------------
+
+The goal of the Smart Contract Instance DID is to provide meta-data about the contract instance.
+At the moment, it contains an account address of the initialization transaction sender, and the list of the contract's entrypoints.
+
+The Smart Contract Instance DID Document MUST contain the following data:
+
+- ``id`` - the DID of the smart contract instance.
+- ``creator`` - a DID of an account that initialized the contract instance represented as a JSON object containing fields ``id`` and ``account``.
+- ``entrypoints`` - a list on the contract's entrypoints. Each entrypoint is an object containing fields ``id`` and ``name``.
+
+The document MAY include any other public data of a smart contract instance.
+
+.. code-block:: json
+
+  {
+    "id": "did:ccd:sci:IND:SUBIND",
+    "owner": {
+      "id": "did:ccd:sci:IND:SUBIND#creator",
+      "account": "did:ccd:NET:acc:ADDR"
+    }
+    "entrypoints": [
+      { "id": "did:ccd:sci:IND:SUBIND#entrypoint-issuerKeys",
+        "name": "issuerKeys"
+      },
+      { "id": "did:ccd:sci:IND:SUBIND#entrypoint-revocationKey",
+        "name": "revocationKey"
+      }
+    ]
+  }
+
+Where ``IND`` and ``SUBIND`` are the contract index and subindex.
+``NET`` and ``ADDR`` correspond to the network and to the owner's account address.
+
+
+.. _concordium-did-pkc:
+
+Public Key Cryptography DID
+---------------------------
+
+The goal of the Public Key Cryptography DID is to represent a public key and the corresponding signature verification method.
+
+The Public Key Cryptography DID Document MUST contain the following data:
+
+- ``id`` - the DID of the public key.
+- ``verificationMethod`` - specifies a `DID verification method <did-vefication-method_>`_ for verifying a signature corresponding to the public key.
+- ``authentication`` - authentication method for the key.
+
+.. code-block:: json
+
+  {
+    "id": "did:ccd:pkc:XX",
+    "verificationMethod": [
+      {
+        "id": "did:ccd:pkc:XX#key-0",
+        "type": "Ed25519VerificationKey2020",
+        "controller": "did:ccd:NET:pkc:PK",
+        "publicKeyMultibase": "fXX"
+      }
+    ],
+    "authentication": [
+      {
+        "did:ccd:pkc:XX#key-0"
+      }
+    ]
+  }
+
+Identity Provider DID
+---------------------
+
+The goal of the Identity Provider DID is identify a Concodrium identity provider (IDP).
+An identity provider is an organization, approved by Concordium, that performs off-chain identification of users.
+IDPs are used in the account creation process to issue an identity.
+IDP DIDs can represent an issuer of a verifiable credential.
```

Comment:
I recommend spelling out identity providers.

I think acronyms in this case are needless

Reply:
Fixed.